### ISSUE TITLE
Broker is a required option

### ISSUE THREAD

**Issue by rafaeljesus** [orpiske/msg-perf-tool#1]
I am running in terminal (mac) the following command:

```sh
src/runner/mpt-runner.sh -b amqp://user:pass@foo.net:5672/event_bus/my.queue -d 10 -p 4 -s 256 -n "sample test"
```

I the following warning appears:
```sh
-e USAGE: ./src/runner/mpt-runner.sh [options]
 -l 'logdir' -- log directory
 -b 'broker-url' -- broker url (ie: amqp://hostname:5672/queue.name)
 -d 'duration' -- test duration (in minutes)
 -c 'count' -- message count
 -C 'config' -- loader configuration
 -s 'size' -- message size (in bytes [default = 1024])
 -p 'parallel count' -- the number of parallel senders and consumers
 -u 'database url' -- (optional) a URL for the elastic database that stores the test data
 -o 'output directory' -- output directory for the test report
 -n 'test name' -- test name
 -t 'throttle' -- throttle (sends messages in a fixed rate [ msgs per second per connection])
 -T 'config-test' -- test case configuration
 -R 'test-run' -- test run
 -h -- this help
```

Any thoughts?

**Comment by orpiske** [comment]
Are you using the code on HEAD or the one packaged on COPR? The runner on HEAD is likely broken right now because I am working on the new version. It has a couple of new features but the options have changed. 

If you are on HEAD, would you please try with the one on branch 0.1.x? Otherwise, just let me know and I will investigate further.

**Comment by orpiske** [comment]
You may also try using the code on 0.1.x-devel branch. I have done a couple of fixes there, although I have not updated the package for that. In particular, I think you might be hitting this one here: https://github.com/orpiske/msg-perf-tool/commit/fe9155e624d9626feb3afc844cc7b5f325e711db.

**Comment by rafaeljesus** [comment]
@orpiske Thank you for your quick reply!

I tried both suggestions but none worked 😕

**Comment by orpiske** [comment]
@rafaeljesus no problem at all. I will investigate this shortly and will share my findings soon. I have done a couple of fixes a few weeks ago for a sample test run and I may have missed something.

**Comment by rafaeljesus** [comment]
@orpiske Thank you mate!

**Comment by orpiske** [comment]
@rafaeljesus I have made a couple of fixes. Would you, please, test them and let me know if it worked for you? The fix is in 0.1.x-devel branch. 

The steps are quite straightforward: 

1. Configure the build: mkdir build && cd build && cmake -DAMQP_SUPPORT=On -DSTOMP_SUPPORT=OFF .. 
2. Build the code: make -j2 
3. Run the loader: cd target/bin ; ./mpt-runner.sh -b amqp://user:pass@host:5672/my.queue -p 4 -d 1 -n -s 256 "sample test" -l ~/tmp 

The log directory is, indeed mandatory, as the sender/receiver log data to a file by default (improvements on this regard are under development on HEAD).

After that, could you please let me know if it worked?

**Comment by orpiske** [comment]
Fixed w/ 1.2.x
